src/App.vue

```diff
@@ -1,8 +1,27 @@
 <script setup>
+import { onMounted, computed } from 'vue'
+import { useStore } from 'vuex'
+import { loadConfig } from './utils/api'
+
+const store = useStore()
+const loadedConfig = computed(() => store.state.isConfigLoaded)
+
+// invoking loadConfig when app first mounts
+onMounted(async () => {
+  const config = await loadConfig()
```

Comment:
nitpick - we're probably only going to use loadConfig here right? if so I would say it doesn't necessarily need to be a util function. especially in the util/api section, since it's not calling an API. if it's specific to loading config on app initialization you could just put the logic inline here.

Reply:
aah okey dokes!

Reply:
fixed!